### PR TITLE
fix: index Oracle.respond internal contract calls

### PR DESCRIPTION
**Pull request by sborrazas**
refs #468

### CODE REVIEW

**Review by thepiwo — APPROVED** [review]

**Review by jyeshe — APPROVED** [review]

**Comment by jyeshe** [comment]
Only compatibility with previous test cases.

**Comment by sborrazas** [comment]
@jyeshe added the migration to include these internal calls as well, feel free to review it again. thanks

**Review by jyeshe — COMMENTED** [review]
migration works only some refactor opportunities...

File: priv/migrations/20220124162300_oracle_respond_internal_calls.ex [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  Writes the internal transfers for the oracle response internal calls.
+  """
+  @spec run(boolean()) :: {:ok, {non_neg_integer(), non_neg_integer()}}
+  def run(_from_start?) do
+    begin = DateTime.utc_now()
+
+    internal_calls_spec =
+      Ex2ms.fun do
+        Model.fname_int_contract_call(index: {"Oracle.respond", call_txi, local_idx}) ->
+          {call_txi, local_idx}
```
[the remark this answers is not in the document]
Comment (sborrazas) [reply priv/migrations/20220124162300_oracle_respond_internal_calls.ex:23]: `local_idx` is actually used [here](https://github.com/aeternity/ae_mdw/pull/480/files#diff-33c6f07aa5c098045ab6db68db177a0bb3f9afcf4ffd20df3b01b6cc303f0982R34)

File: priv/migrations/20220124162300_oracle_respond_internal_calls.ex [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+            OracleResponseMutation.new(block_index, call_txi, oracle_pk, fee)
+          ]
+        rescue
+          # TreeId = <<OracleId/binary, QId/binary>>,
+          # Serialized = aeu_mtrees:get(TreeId, Tree#oracle_tree.otree)
+          # raises error on unexisting tree_id
+          error ->
+            Log.error(error)
+            []
+        end
```
[the remark this answers is not in the document]
Comment (sborrazas) [reply priv/migrations/20220124162300_oracle_respond_internal_calls.ex]: Done, thanks!

**Pull request merged**